src/schema/v2/homeView/featuredFairsResolver.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLFieldResolver } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { HomePageFairsModuleType } from "../home/home_page_fairs_module"
```

Comment:
**thought (non-blocking):** Interesting because we've not yet had to import from the existing `/home/home_page_*_module` resolvers. I don't think it's necessarily a problem, but presents a small chore in the future if/when we retire and clean up the existing `HomePage` type and its fields/resolvers.

Reply:
Agree with you, we will have to transfer all logic from there